### DISCUSSION TITLE
how to receive a multiple data record in one table column

### DISCUSSION THREAD

**Discussion by razisek** [filamentphp/filament#2732] [Help]
I will process a multiple data but in one column, how to?

**Comment by otavio-araujo** [comment]
You can do View Columns or Custom Columns:

https://filamentphp.com/docs/2.x/tables/columns#view-column

https://filamentphp.com/docs/2.x/tables/columns#building-custom-columns

**Reply by razisek** [reply]
how to get data other column?

**Reply by danharrin** [reply]
`{{ $getRecord()->column_name }}`
